examples/realworld/ext/article/components/ArticleViewPage.js

```diff
@@ -84,6 +83,7 @@ const ArticleViewPage = (props) => {
     if (!window.confirm('Are you sure you want to delete the article?')) return
     try {
       await deleteArticle({ id: article.id })
+      history.push('/')
```

Comment:
Why did we give up on this? It happens automatically?

Reply:
We didn't give up (the PR adds this line). I added it in precisely because it doesn't happen automatically :smile: